apps/ledger-live-desktop/src/renderer/drawers/SwapOperationDetails/index.tsx

```diff
@@ -128,6 +130,14 @@ const SwapOperationDetails = ({
     fromCurrency.type === "CryptoCurrency" &&
     getTransactionExplorer(getDefaultExplorerView(fromCurrency), operation.hash);
 
+  useEffect(() => {
+    const getProvideData = async () => {
+      const data = await getSwapProvider(provider);
```

Comment:
What should happen if the getSwapProvider request throws?

Reply:
There is a fall back in the getSwapProvider with static data